src/gennylib/include/gennylib/Orchestrator.hpp

```diff
@@ -107,7 +107,7 @@ class Orchestrator {
     PhaseNumber _current = 0;
 
     // _phaseNumberGauge should keep up to date with _current.
-    metrics::Gauge _phaseNumberGauge;
+    metrics::v1::Gauge _phaseNumberGauge;
```

Comment:
Let's just kill `_phaseNumberGauge`. We'll probably want something similar in the future but I think it really wants to be a custom gauge within metrics::Operation (requires TIG-1420).

Reply:
Actually `timer()`, `gauge()` and `counter()` methods on `ActorContext` can probably just be removed now, ya?